_posts/2021-03-14-curating/curating.Rmd

```diff
@@ -0,0 +1,329 @@
+---
+title: "Curating for @WeAreRLadies on Twitter"
+description: |
+  From creating content to cultivating connections 
+base_url: https://www.pipinghotdata.com/
+preview: img/arno-smit-iI72r3gSwWY-unsplash.jpg
+twitter:
+  site: "@PipingHotData"
+  creator: "@PipingHotData"
+date: 03-14-2021
+author:
+  - name: Shannon Pileggi
+output:
+  distill::distill_article:
+    toc: true
+    toc_float: true
+    toc_depth: 1
+    self_contained: false
+draft: false
+creative_commons: CC BY
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+```{r}
+folder <- "2021-03-14-curating"
+```
+
+```{r}
+library(tidyverse)
+library(ggtext)
+library(ggiraph)
+library(tweetrmd)
+```
+
+
+```{r, fig.cap="Photo by Arno Smit on Unsplash (https://unsplash.com/photos/iI72r3gSwWY)", fig.alt="Image of open door to porch with inviting sitting chair, surrounded by plants."}
+knitr::include_graphics("img/arno-smit-iI72r3gSwWY-unsplash.jpg")
+```
+
+# TL; DR
+
+In February 2021 I tweeted to a daunting >20k followers by curating for [`@WeAreLadies`](https://twitter.com/WeAreRLadies){target="_blank"} on Twitter. This was great opportunity to share knowledge, connect with others, and learn something in return, ultimately cultivating new connections and collaborations. From strategy to fruition, I hope this post helps you confidently enroll as a curator!
```

Comment:
```suggestion
In February 2021 I tweeted to a daunting >20k followers by curating for [`@WeAreLadies`](https://twitter.com/WeAreRLadies){target="_blank"} on Twitter. This was great opportunity to share knowledge, interact with others, and learn something in return, ultimately cultivating new connections and collaborations. From strategy to fruition, I hope this post helps you confidently enroll as a curator!
```
(the 'connect' made it a bit repetitive with the later comment 'new connections and collaborations')

Reply:
good catch, thank you!